Server/API/login/postLogin.go

```diff
@@ -0,0 +1,48 @@
+package login
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/google/uuid"
+	"github.com/labstack/echo/v4"
+	entities "ishtaloo.io/Entities"
+)
+
+func postLogin(c echo.Context) (err error) {
+	cookie := new(http.Cookie)
+	cookie.Name = "userId"
+	u := new(entities.LoginRequest)
+	if err = c.Bind(u); err != nil {
+		return err
+	}
+	id := uuid.New()
```

Comment:
@y-golde review this new part please, I started using uuid package

Reply:
approved